<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QToDoListClass</class>
 <widget class="QMainWindow" name="QToDoListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QToDoList</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8,3">
    <item>
     <widget class="QScrollArea" name="scrollAreaMenus">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetMenus">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>136</width>
         <height>701</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelPageTitle">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>我的首页</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>288</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnRefresh">
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSort">
          <item>
           <property name="text">
            <string>按到期时间排序</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按创建时间排序</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按优先级排序</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineAddItem">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>+点击新建待办</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollAreaItemList">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>548</width>
           <height>640</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="SideBarWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SideBarWidget</class>
   <extends>QWidget</extends>
   <header>sidebarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qtodolist.qrc"/>
 </resources>
 <connections/>
</ui>
